src/components/Book.jsx

```diff
@@ -1,23 +1,49 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import './styles/book.css';
 
 const Book = (props) => {
-  const { author, title, id, handleDeleteProps } = props;
+  const { author, category, title, id, handleDeleteProps } = props;
   return (
-    <li className="bookContainer">
-      <p>
-        {title}
-        ::
-        {author}
-      </p>
-      <button
-        type="button"
-        data-id={id}
-        onClick={(event) => handleDeleteProps(event.target.dataset.id)}
-      >
-        delete
-      </button>
+    <li className="book-panel">
+      <div className="leftblock">
+        <span className="book-genre">{category}</span>
+        <span className="book-title">{title}</span>
+        <span className="book-author">{author}</span>
+        <span className="leftblockbuttons">
+          <button data-id={id} type="button" className="lbBtn">
+            Comments
+          </button>
+          <button
+            data-id={id}
+            type="button"
+            className="lbBtn"
+            onClick={(event) => handleDeleteProps(event.target.dataset.id)}
+          >
+            Remove
+          </button>
+          <button data-id={id} type="button" className="lbBtn">
+            Edit
+          </button>
+        </span>
+      </div>
+
+      <div className="container">
+        <div className="middleblock">
+          <div className="progress-calc" />
+          <span>
+            <div className="progress">64%</div>
+            <div className="completed">Completed</div>
+          </span>
+        </div>
+
+        <div className="rightblock">
+          <div className="chapter">CURRENT CHAPTER</div>
+          <div className="lesson">Chapter 309</div>
+          <button type="button" className="UpdateData">
+            UPDATE PROGRESS
+          </button>
```

Comment:
- [ ] Please remove the border of `'UPDATE PROGRESS'` button and use the correct width and height to make sure the button text is visible on the same line.

### Your design

![image](https://user-images.githubusercontent.com/83803414/172880951-c4179a64-6f80-45d3-92e6-9e7ad041a7b4.png)

### Zeplin design 

![image](https://user-images.githubusercontent.com/83803414/172881091-b58f8294-ab5f-4d19-85f2-4de7b95dc32e.png)